### PR TITLE
[stable/kibana] Set hosts as env var only

### LINKED ISSUE #14062: [stable/kibana] FATAL  ValidationError: child "elasticsearch" fails because ["hosts" is not allowed]
Exactly the same as : https://github.com/elastic/kibana/issues/32303#issue-416128042
We are using Kibana 6.5.4, therefor as stated in the values.yaml we have swapped the elasticsearch.hosts for .url, still to no effect.

**Version of Helm and Kubernetes**: K8s 1.13.3

**Which chart**:
stable/kibana

**What happened**:
Kibana pod reports:
`FATAL  ValidationError: child "elasticsearch" fails because ["hosts" is not allowed]`

`{"type":"error","@timestamp":"2019-05-23T06:15:39Z","tags":["fatal","root"],"pid":1,"level":"fatal","error":{"message":"child \"elasticsearch\" fails because [\"hosts\" is not allowed]","name":"ValidationError","stack":"ValidationError: child \"elasticsearch\" fails because [\"hosts\" is not allowed]\n`

**What you expected to happen**:
elasticsearch.url: elasticsearch-client:9200
to be accepted as a string

**How to reproduce it** (as minimally and precisely as possible):
helm install --name kibana stable/kibana (6.5.4 with the proper values edited)
kubectl logs kibana

### PR DESCRIPTION
**Pull request by naseemkullah**
Signed-off-by: Naseem <naseemkullah@gmail.com>

#### What this PR does / why we need it:

setting `elasticsearch.hosts` by default in `kibana.yml` causes issues if pinning kibana to version < 6.6

This was brought up here https://github.com/helm/charts/issues/14062#issue-447128407

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #14062

#### Special notes for your reviewer:

cc @monotek @desaintmartin @Towmeykaw @mjrepo2

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by Towmeykaw** [comment]
Did a quick test and it looks good to me.

File: stable/kibana/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: kibana
-version: 3.0.0
+version: 3.1.0
```
Comment (monotek) [inline stable/kibana/Chart.yaml]: Is this a breaking change for people who used 3.0 before?

**Review by monotek — COMMENTED** [review]
I'm currently thinking about to use a value to create both configs (hosts & url) as env vars.
This should work for all versions without the need to change the config.

@compleatang 
Whats your opinion on this?

**Comment by compleatang** [comment]
@monotek I think that would be nice. I kinda feel like we're going round and round in circles on this env var and whether the config should have defaults or be empty. I'm underwater this week but it would be nice to have a more sustainable solution in my view.

File: stable/kibana/Chart.yaml (hunk 1 above)
Comment (naseemkullah) [reply stable/kibana/Chart.yaml]: No, I'll change it to `3.0.1`

**Comment by compleatang** [comment]
@naseemkullah I'm sorry about the delay in processing the chart PR. Thanks for the contribution. Unfortunately we're going to put a hold on this issue for the reasons below.

In combination with the elastic team, we've decided to deprecate stable/kibana going forward in favor of elastic/kibana. You can see the tracking issue for the work here -> https://github.com/helm/charts/issues/14935

We are encouraging all open issues to contribute their work to the "new" community chart led by the elastic team here -> https://github.com/elastic/helm-charts/tree/master/kibana

If you have any questions please feel free to comment on #14935 

/hold

**Comment by naseemkullah** [comment]
Thanks @compleatang FYI I've made a similar PR to elastic/kibana.

**Pull request closed without merging**